browser-based-ingestion/README.md

```diff
@@ -0,0 +1,249 @@
+### KVS Browser based Ingestion is used to stream video from the system's webcam to the KVS backend directly without any proxies. ###
+
+Currently, it is only available via Chrome. 
+There are four important tasks that this repository helps perform
+1. Obtaining the video from the webcam/ uploading video file
+2. Converting it to containers and codecs expected by the playback
+3. Sending to the putMedia API via POST calls
+4. Viewing the stream on the browser
+
+
+### Setup ###
+To set-up the project and run the example index.html and index.js files, follow the steps below:
+1. Create a webpack project referring to https://webpack.js.org/guides/getting-started/
+    ```
+    mkdir webpack-demo
+    cd webpack-demo
+    npm init -y
+    npm install webpack webpack-cli --save-dev
+    ```
+
+2. Create dist and src folders in webpack-demo
+    ```
+    mkdir dist
+    mkdir src
+    ```
+
+3. Add webpack.config.js with the following content to the src folder
+    ```javascript
+    const HtmlWebPackPlugin = require('html-webpack-plugin')
+    module.exports = {
+    target: 'node',
+    entry: './dist/index.js',
+    output: {
+        filename: 'main.js'
+    },
+    module: {
+        rules: [
+        {
+            test: /.html$/,
+            use: [
+            {
+                loader: 'html-loader',
+                options: { minimize: true }
+            }
+            ]
+        }
+        ]
+    },
+    plugins: [
+        new HtmlWebPackPlugin({
+        template: './src/index.html',
+        filename: './index.html'
+        })
+    ]
+    }
+    ```
+
+4. Install aws-sdk and aws4
+    ```
+    npm install aws-sdk -save--dev
+    npm install aws4 --save-dev
+    ```
+
+5. Now you can either create your own HTML page or use the one in the dist folder of this project.  
+    If you create your own HTML page, then in order to use the functions from this project add scripts (i) and (ii) from below to your index.html in the dist folder. To use your own index.js from src folder, add script (iii) to your index.html in the dist folder 
+    i. `<script src="https://unpkg.com/@ffmpeg/ffmpeg@0.8.3/dist/ffmpeg.min.js"></script>` 
+    ii. `<script src="https://www.WebRTC-Experiment.com/RecordRTC.js"></script>` 
+    iii. `<script src="main.js"></script>`
+
+6. Now, add AWS-SDKCalls.js, createStream.js, getDataEndpoint.js, getProcessedVideo.js from this project's dist folder to yours, so that you can use the functions from the same in your index.js. (You can add index.js from this project's dist folder to yours to run the example)
+
+7. Once you have imported all the scripts as per directions above and have index.html in dist and index.js in src, run `npx webpack` from webapck-demo folder
+
+8. You can run the project on locahost using http-server
+    ```
+    brew install http-server
+    http-server -p 8080 -o
+    ```
+
+
+
+### Functions in the project ###
```

Comment:
A better heading would be `Utilities` or `Code documentation`. And then you can define this way: 

1. Method: `getBlobFromWebcam (<whatever args it takes, if any>)
2. Method: `getBlobFromFile (<whatever args it takes, if any>)
3. ...... so on

Reply:
Thanks! I have incorporated the changes